lib/configs/clientLoaders.js

```diff
@@ -0,0 +1,21 @@
+module.exports = [
+  'esnextreact',
+  'json',
+  'css',
+  // Enable less and tell webpack that our static asses all live in <project>/assets
+  {
+    generator: 'less',
+    assetsDirectory: 'assets',
+    publicPath: './',
+  },
+  // resolves webfonts urls, assumes they all live in <project>/assets/fonts
+  {
+    test: /\.(eot|woff|ttf|svg)(\?.+)?$/,
+    loader: 'file?emitFile=false&name=fonts/[name].[ext]',
+  },
+  // resolves images as urls, assumes they all live in <project>/assets/img
+  {
+    test: /\.(png|svg|jpe?g|gif|tiff)$/,
```

Comment:
Actually, you were right, I updated the regexp to make sure the directory is what the emitted url expects.